examples/next-typescript/pages/checkbox.tsx

```diff
@@ -0,0 +1,102 @@
+import { Checkbox } from "@tonightpass/kitchen";
+import { NextPage } from "next";
+import React from "react";
+
+const CheckboxPage: NextPage = () => {
+  return (
+    <>
+      <div
+        style={{
+          display: "flex",
+          flexDirection: "column",
+          alignItems: "center",
+          justifyContent: "center",
+          height: "100vh",
+          gap: "10px",
+        }}
+      >
+        <p>default</p>
+        <div
+          style={{
+            display: "flex",
+            gap: "16px",
+            alignItems: "center",
+            justifyContent: "center",
+          }}
+        >
+          <Checkbox>Default</Checkbox>
+        </div>
+
+        <p>checked</p>
+        <div
+          style={{
+            display: "flex",
+            gap: "16px",
+            alignItems: "center",
+            justifyContent: "center",
+          }}
+        >
+          <Checkbox checked>Checked</Checkbox>
+        </div>
+
+        <p>no label</p>
+        <div
+          style={{
+            display: "flex",
+            gap: "16px",
+            alignItems: "center",
+            justifyContent: "center",
+          }}
+        >
+          <Checkbox />
+        </div>
+
+        <p>disabled</p>
+        <div
+          style={{
+            display: "flex",
+            gap: "16px",
+            alignItems: "center",
+            justifyContent: "center",
+          }}
+        >
+          <Checkbox disabled>Disabled</Checkbox>
+        </div>
+
+        <p>overflowing text</p>
+        <div
+          style={{
+            display: "flex",
+            gap: "16px",
+            alignItems: "center",
+            justifyContent: "center",
+          }}
+        >
+          <Checkbox>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+            eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
+            ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+            aliquip ex ea commodo consequat. Duis aute irure dolor in
+            reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
+            pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
+            culpa qui officia deserunt mollit anim id est laborum
+          </Checkbox>
+        </div>
+
+        <p>full width</p>
+        <div
+          style={{
+            display: "flex",
+            gap: "16px",
+            alignItems: "center",
+            justifyContent: "center",
+          }}
+        >
+          <Checkbox fullWidth>Full Width</Checkbox>
```

Comment:
You have to test it in a ``Container`` with a ``maxWidth`` because it is impossible to test...